### PR TITLE
fix:TalComUser Application form acknowledgment and redirection

### PR DESCRIPTION
**Pull request by jeyanthi-sm**
TalComUser Applicant form acknowledgment and redirection

### CODE REVIEW

File: apps/dev-recruiters/src/pages/second.tsx [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+  const handleCloseModal = (): void => {};
+
+  return (
     <BoxContainer>
       {/* <TalcommunityPage></TalcommunityPage> */}
       <HeaderTalPage></HeaderTalPage>
-      <SignUpForm />
-      {/* <TalFormPage></TalFormPage> */}
+      <SignUpForm handleCloseModal={handleCloseModal} />
+      {/* <TalCommForm></TalFormPage> */}
```
Comment (ajaywayase) [inline apps/dev-recruiters/src/pages/second.tsx:20]: is this commented part required to keep?

Comment (jeyanthi-sm) [reply apps/dev-recruiters/src/pages/second.tsx:20]: It was commented before and it was not part of this change.

**Review by ajaywayase — APPROVED** [review]

File: apps/dev-recruiters/src/components/modules/TalcommunityPage/TalFormPage/signUpForm.tsx [hunk 2]
```diff
@@ -157,13 +158,6 @@ export default function TalCommForm({ handleCloseModal, position }: Props) {
                       onChange={handleSetCheckCheckbox}
                       required
                     />
-                    <Checkbox
-                      label="I have read and agreed to the terms and conditions"
```
Comment (Enjoy2Live) [inline apps/dev-recruiters/src/components/modules/TalcommunityPage/TalFormPage/signUpForm.tsx:161]: Why remove this checkbox? I think it's nice to have it.
Applying without agreeing to the terms and conditions might become problematic down the line.

**Review by Enjoy2Live — CHANGES_REQUESTED** [review]

File: apps/dev-recruiters/src/components/modules/TalcommunityPage/TalFormPage/signUpForm.tsx (hunk 2 above)
Comment (jeyanthi-sm) [reply apps/dev-recruiters/src/components/modules/TalcommunityPage/TalFormPage/signUpForm.tsx:161]: After the review, I added the Terms and Conditions checkbox and reverted the commented lines on the 'Second' page to their previous state.

**Pull request merged**